### PR TITLE
changed how worksheet title and  web cli looks

### PR DESCRIPTION
**Pull request by Levilian**
#1520 : made cli more transparent
#1532 : made navbar above the web cli, which is hidden by default
#1548 : changed how title and name look like and their respective hover behavior

### CODE REVIEW

File: frontend/src/css/imports.css [hunk 1]
```diff
@@ -7736,9 +7737,16 @@ fieldset[disabled] .worksheet_content .header-row .controls .edit-features .btn-
   }
   #ws_search,
   #worksheet .ws-panel,
-  #worksheet .dragbar {
+  #terminal-button {
```
Comment (percyliang) [inline frontend/src/css/imports.css:7740]: We should really get off of this css file soon before we start hacking this too much...

**Comment by epicfaace** [comment]
Oh, it appears this PR fixes #1549 as well.

**Review by epicfaace — COMMENTED** [review]
Looks good.

I think you should probably keep the "dashboard" and "Codalab Dashboard" text in a blue color, though (just remove the underline upon hover so it doesn't look like a link). Otherwise, the text blends in with other text around it and it's not obvious that it's editable -- how would the user know that "dashboard" is editable, but "by codalab" isn't?

image

File: frontend/src/css/imports.css [hunk 2]
```diff
@@ -6743,18 +6744,18 @@ body,
   position: relative;
   height: calc(100% - 20px - 25px);
 }
-#worksheet .editable-field:hover {
+.editable-field:hover {
```
Comment (epicfaace) [inline frontend/src/css/imports.css:6747]: When you hover on an editable field, the text moves slightly off around 1px. Can you fix this?

File: frontend/src/css/imports.css [hunk 3]
```diff
@@ -6743,18 +6744,18 @@ body,
...
+  border-radius: 2px;
 }
 #ws_search {
   position: fixed;
   width: 100%;
   margin: 0;
   padding: 10px 0;
   text-align: center;
-  background-color: #F8F9FA;
+  background: rgba(245, 245, 245, 0.55);
```
Comment (epicfaace) [inline frontend/src/css/imports.css]: Can you make this a bit more opaque? Right now it's too transparent and a bit hard to read:

image

**Review by epicfaace — CHANGES_REQUESTED** [review]

File: frontend/src/css/imports.css (hunk 1 above)
Comment (Levilian) [reply frontend/src/css/imports.css:7740]: I see. In terms of hiding CLI when the screen is smaller than a certain size, is that expected behavior? Should I remove this restriction completely? 

Right now what I'm doing here is to hide the Open/Hide Terminal toggle bar when the screen becomes too small.

**Pull request merged**